SouvlakMVP/SouvlakMVP/Graph.cs

```diff
@@ -0,0 +1,553 @@
+﻿namespace SouvlakMVP;
+
+using System.Numerics;
+using System.Linq;
+using indexT = System.Int32;
```

Comment:
My suggestions - so that it is clear that it's for vertices (also change in other places!):
```suggestion
using indexVertexT = System.Int32;
```
```suggestion
using vertexIndexT = System.Int32;
```